css/index.css

```diff
@@ -0,0 +1,888 @@
+html {
+	-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+body {
+    margin: 0;
+    background-color: #ffffff;
+    color: #483C32;
+    font-family: 'Preahvihear', sans-serif;
+    font-weight: 400;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+*::selection {
+	background-color: #828c7f;
+	background-color: rgba(196, 226, 226, 1.00);
+}
+
+
+/*********************** HEADER *****************/
+
+.container {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1000px;
+}
+
+header {
+    padding: 10px 20px;
+    background-color: rgba(229, 231, 233, 0.3);
+}
+
+header nav {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
```

Comment:
Since you're so far advanced with what you're doing on this project, you could also look into "pinning" your navbar to the top of the page, so it's always visible as you scroll down the page:


https://github.com/pstnv/freyr-intro/assets/12771651/321be807-f97a-49dd-bf7b-6228a13b5dbf

You can see here my change still cuts off the very top of each section, so you'd have to do some whitespace adjustment too!